autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Approximately how long does this Autopilot job take to run? I see it is sleeping in 30-second increments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I've seen it run anywhere from 20 to 80 minutes.  Suggestions?</p>

Reply:
<p>I think it's helpful for the customer to know that. Are there certain specific factors that influence the runtime, or it's just variable? Something like <code>This job may take 20-80 minutes to complete</code> or similar. </p>

Reply:
<p>Ok.  Added a comment.  It seems to vary with factors a user can't control.  The most likely ones to me are things in the ML method AutoPilot uses as well as uncertainty related to provisioning times in the platform.  Both are intentionally opaque to the user in AutoPilot.</p>